test/OpenTelemetry.Instrumentation.AspNetCore.Tests/BasicTests.cs

```diff
@@ -127,6 +127,15 @@ public async Task SuccessfulTemplateControllerCallUsesParentContext()
             Assert.Equal(2, activityProcessor.Invocations.Count); // begin and end was called
             var activity = (Activity)activityProcessor.Invocations[1].Arguments[0];
 
+#if !NETCOREAPP2_1
```

Comment:
nit: probably swap the if condition and `if/else` and check `NETCOREAPP2_1` which looks more natural.

Reply:
agree. saving it for a next pr, as merging this is required to validate if this fixes a bug we are after